typings/webgme.d.ts

```diff
@@ -1574,7 +1681,7 @@ declare namespace GmeClasses {
          * @return a dictionary where every the key of every entry is an absolute path of a set owner node. 
          * The value of each entry is an array with the set names in which the node can be found as a member.
          */
-        isMemberOf(node: Core.Node): Core.DataObject;
+        isMemberOf(node: Core.Node): object;
```

Comment:
I would use a better return value. Perhaps something like
```
type MemberSets = {[ownerPath: string]: string[]};
```
Not great but better than an opaque object.